erpnext/education/doctype/fees/fees.py

```diff
@@ -107,6 +110,52 @@ def make_gl_entries(self):
 		from erpnext.accounts.general_ledger import make_gl_entries
 		make_gl_entries([student_gl_entries, fee_gl_entry], cancel=(self.docstatus == 2),
 			update_outstanding="Yes", merge_entries=False)
+	
+	def check_for_individual_heads(self):
+		linked_fs = frappe.get_doc("Fee Structure", self.fee_structure)
```

Comment:
You don't need to get the entire document to check the value. It'll be faster to just do:

```
return frappe.db.get_value("Fee Structure", self.fee_structure, "process_fees_category_account_heads")
```

Since the value will be `False` if the box is unchecked.